examples/with-react-intl/server.ts

```diff
@@ -21,7 +22,10 @@ Promise.all([app.prepare(), ...SUPPORTED_LOCALES.map(polyfill)]).then(() => {
     const accept = accepts(req);
     const locale = accept.language(supportedLanguages) || 'en';
     (req as any).locale = locale;
-
+    const nonce = crypto.randomBytes(20).toString('hex');
+    (req as any).nonce = nonce;
+    // TODO: This will blow up other next inline JS but next.js should prob fix this
```

Comment:
@timneutkens @lfades I think u guys should support `nonce` it's a pretty big security safeguard against extension malwares